source/uwp/AdaptiveCardTestApp/Expected/cortana-skills.Tests.Action.CustomParsing.json

```diff
@@ -1 +1 @@
-{"HostConfigHash":"a60a3b3","CardHash":"06e61b8","Error":null}
\ No newline at end of file
+{"HostConfigHash":"8a51ac6","CardHash":"06e61b8","Error":null}
```

Comment:
Since this changes all the images in UWP to use the wrong fonts, i'd hold off on checking this in until we've got UWP support. I'm working on it now, and will send a PR to this branch when it's ready